### DISCUSSION TITLE
Don't automatically scroll to the top when switching to a pinned tab

### DISCUSSION THREAD

**Discussion by DerRockWolf** [piroor/treestyletab#3427] [General]
Is it possible to configure TST to don't automatically scroll to the top of the tree when switching to a pinned tab?
I searched through the settings but couldn't fine it.

In my opinion disabling the automatic scrolling would be more aligned to how the default firefox tab selection is working.

**Comment by piroor** [comment]
It is impossible for now, thus I've added a new secret option `syncActiveStateToBundledTabs `.
https://github.com/piroor/treestyletab/commit/8c1b093f9b50ab158d360fbff3ccf9d1a50520b9
TST options => Development => All Configs => unchecking syncActiveStateToBundledTabs will disables scrolling and expansion for such cases.

**Reply by DerRockWolf** [reply]
Great, thank you so much for implementing this :heart: 
I've tested it and can confirm that it does exactly what I expect.

Looking forward to the next release :slightly_smiling_face:
